### PR TITLE
PN-202: Improve the look of matching details

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by buske** [comment]
@veronikaslc Could you please include a screenshot of the updated UI?

**Comment by buske** [comment]
How does this behave if the gene has variants in the two cases? Should that section have the headings as well, or will that further confuse things because then it looks like WNK1 is only in the current patient?

**Comment by veronikaslc** [comment]
`course.phenomecentral.org` is updated with changes, you can test the look there

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 1]
```diff
@@ -684,7 +684,7 @@
 
       var row = this._getEmptyTableRow(modeOfInheritanceTable);
 
-      var handleEmptyArray = function(moiArray) {return (moiArray &amp;&amp; moiArray.length &gt; 0) ? moiArray : ['-'];};
+      var handleEmptyArray = function(moiArray) {return (moiArray &amp;&amp; moiArray.length &gt; 0) ? moiArray : ['Unknown'];};
```
Comment (sdumitriu) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Doesn't this need to be translated?

**Comment by veronikaslc** [comment]
@sdumitriu Added translations

**Pull request merged**